### PR TITLE
Fix for #1039

### PR DESCRIPTION
**Pull request by nekosaur**

### CODE REVIEW

File: src/mixins/resizable.js [hunk 1]
```diff
@@ -14,6 +14,10 @@ export default {
 
   mounted () {
     window.addEventListener('resize', this.debouncedOnResize, { passive: true })
+
+    this.$vuetify.load(() => {
```
Comment (johnleider) [inline src/mixins/resizable.js]: this.$vuetify.load(this.onResize)

File: src/util/load.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ function load (cb, i = 0) {
   }
 
   document.addEventListener('DOMContentLoaded', cb)
+  window.addEventListener('load', cb)
```
Comment (johnleider) [inline src/util/load.js]: merge in master, this file has changed

**Comment by johnleider** [comment]
Is committing of the dist files necessary with this?

**Comment by nekosaur** [comment]
Didn't even think about that. It's because I merged in master

**Pull request merged**